### PR TITLE
Fixed remaining a11y violations for account console

### LINKED ISSUE #22670: Account Console: Remaining A11y Issues
### Before reporting an issue

- [X] I have searched existing issues
- [X] I have reproduced the issue with the [latest nightly release](https://github.com/keycloak/keycloak/releases/tag/nightly)

### Area

account/ui

### Describe the bug

Critical and serious issues were detected in the account ui that need fixing.

### Version

22.0.1

### Expected behavior

Violations are fixed.

### Actual behavior

Violations can be detected using the Deque browser extension.

### How to Reproduce?

Navigate to Application and Account Security/Signing in pages and use the Deque browser extension to scan and detect violations. 

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by agagancarczyk**
Closes https://github.com/keycloak/keycloak/issues/22670

### CODE REVIEW

File: js/apps/account-ui/src/applications/Applications.tsx [hunk 1]
```diff
@@ -154,8 +155,9 @@ const Applications = () => {
             </DataListItemRow>
 
             <DataListContent
+              id={`content-${application.clientId}`}
               className="pf-u-pl-4xl"
-              aria-label={t("applicationDetails")}
+              aria-label={`${t("applicationDetails")}-${application.clientId}`}
```
Comment (jonkoops) [inline js/apps/account-ui/src/applications/Applications.tsx:160]: An ARIA label should be human readable, if you want to interpolate a string value into a translated text you will have to do so in an expression of the label itself:

```ts
t("applicationDetails", { clientId: application.clientId })
```

**Review by jonkoops — CHANGES_REQUESTED** [review]

File: js/apps/account-ui/src/applications/Applications.tsx (hunk 1 above)
Comment (agagancarczyk) [reply js/apps/account-ui/src/applications/Applications.tsx:160]: I see, thank you @jonkoops !

File: js/apps/account-ui/src/applications/Applications.tsx [hunk 2]
```diff
@@ -154,8 +155,11 @@ const Applications = () => {
             </DataListItemRow>
 
             <DataListContent
+              id={`content-${application.clientId}`}
               className="pf-u-pl-4xl"
-              aria-label={t("applicationDetails")}
+              aria-label={t("applicationDetails", {
+                clientId: application.clientId,
```
Comment (jonkoops) [inline js/apps/account-ui/src/applications/Applications.tsx:161]: This is currently not accepted by the label for "applicationDetails", so you'll have to update it as well, or perhaps define a new label that has `{{clientId}}` inside of it. Either way it has to be human readable and descriptive.

**Review by jonkoops — CHANGES_REQUESTED** [review]

**Review by jonkoops — APPROVED** [review]

**Comment by agagancarczyk** [comment]
Backporting https://github.com/keycloak/keycloak/pull/22766

**Pull request merged**